NitroxModel/Logger/Log.cs

```diff
@@ -303,6 +317,7 @@ public enum LogLevel
         Debug = 1,
         Information = 2,
         Warning = 3,
-        Error = 4
+        Error = 4,
+        Verbose = 5
```

Comment:
Is there a reason you appended it as the last and not before Debug (with 0 as value)?